concepts/modularization/README.md

```diff
@@ -151,7 +152,7 @@ Yes, but under the hood, `kyma-operator` will be used to install component opera
 
 ## I have a simple component with a Helm chart. Why do I need an operator?
 
-With the operator, you can fully control your component lifecycle and ensure that your component are reconciled  to the desired state (watch component config and actual state).
+With the operator, you can fully control your component lifecycle and ensure that your component are reconciled to the desired state (watch component config and actual state). Each operator comes with a custom resource that describes module configuration and represents module installation status. It is a way to enable users with providing chart overrides in a controlled way.
```

Comment:
```suggestion
With the operator, you can fully control your component lifecycle and ensure that your component are reconciled to the desired state (watch component config and actual state). Each operator comes with a custom resource that describes module configuration and represents module installation status. It is a way to enable users with providing chart overrides in a controlled way.
```
I think this is not only used for chart overrides but also for operator-related settings right? For sure, chart overrides are the most straight-forward use case, but also things like namespace injection for istio would be configured here right?